ena-oauth2-profile.md

```diff
@@ -217,6 +219,8 @@ The redirect URIs provided MUST be absolute URIs as defined in section 4.3 of \[
 
 If more than one redirect URI is provided, different domains SHOULD NOT be used.
 
+If a client needs to register to multiple authorization servers it can not re-use a redirect uri that has been used for another authorization server see [5.1.1](#authorization-requests)
```

Comment:
We can't have this requirement since it would make it impossible for a client to work in a federative context. There are other countermeasures for mix-up attacks. Removing this ...